packages/p2p/src/pages/my-profile/my-profile-stats/my-profile-privacy/my-profile-privacy.scss

```diff
@@ -23,23 +23,5 @@
                 margin-top: 0;
             }
         }
-
-        &-button {
-            width: 1.3rem;
-            height: 1.3rem;
-            border-radius: 1.3rem;
-        }
-
-        width: 3.2rem;
-        height: 1.7rem;
-        border-radius: 3.2rem;
-    }
-}
-
-.dc-toggle-switch:checked + .dc-toggle-switch__label .my-profile-privacy__toggle-button {
-    transform: translateX(1.35rem);
-
-    @include rtl {
-        transform: translateX(-1.35rem);
```

Comment:
this won't affect the arabic translations right

Reply:
no, this is handled by ToggleSwitch component